client/src/utils/require-template.js

```diff
@@ -1,12 +1,56 @@
 'use strict';
 
 var format = require('stringformat');
-var templateNotSupported = 'Error loading component: template "{0}" not supported';
+var path = require('path');
+var _ = require('underscore');
```

Comment:
we don't have underscore in the client! Made the same mistake when reviewing the previous PR with @mattiaerre and then realised - are you able to go vanilla with client's code?